frontend/app/brains-management/[brainId]/components/BrainManagementTabs/BrainManagementTabs.tsx

```diff
@@ -1,24 +1,43 @@
 import { Content, List, Root } from "@radix-ui/react-tabs";
+import { useRouter } from "next/navigation";
+import { useState } from "react";
 import { useTranslation } from "react-i18next";
 
+import Button from "@/lib/components/ui/Button";
+import { useBrainContext } from "@/lib/context/BrainProvider/hooks/useBrainContext";
+
 import { BrainTabTrigger, PeopleTab } from "./components";
+import ConfirmationDeleteModal from "./components/Modals/ConfirmationDeleteModal";
 import { SettingsTab } from "./components/SettingsTab/SettingsTab";
 import { useBrainManagementTabs } from "./hooks/useBrainManagementTabs";
 
 export const BrainManagementTabs = (): JSX.Element => {
-  const { t } = useTranslation(["translation", "config"]);
+  const { t } = useTranslation(["translation", "config", "delete_brain"]);
   const { selectedTab, setSelectedTab, brainId } = useBrainManagementTabs();
+  const { deleteBrain, setCurrentBrainId } = useBrainContext();
+  const [isDeleteModalOpen, setIsDeleteModalOpen] = useState(false);
+  const router = useRouter();
 
   if (brainId === undefined) {
     return <div />;
   }
 
+  const handleDeleteBrain = (): void => {
+    void deleteBrain(brainId);
+    setCurrentBrainId(null);
+    router.push("/brains-management");
+    setIsDeleteModalOpen(false);
+  };
+
```

Comment:
Nice if this can be moved to `useBrainManagementTabs.ts`

Reply:
Done